src/docs.md

```diff
@@ -367,6 +367,30 @@ const response = await cadenzaClient.fetchObjectInfo('embeddingTargetId', 'layer
 const objectInfo = await response;
 ```
 
+### Fetch the area intersection from a Workbook Map View Layer
+
+<small>API: [CadenzaClient#fetchObjectInfo(./classes/CadenzaClient.html#fetchAreaIntersection)</small>
+
+Download the intersection area from a workbook map view layer in JSON format for a given area. The result contains all intersecting objects of the layer, their object ID, if defined the object name and a geometry representing the intersection area, with area size and area perimeter.
```

Comment:
In Cadenza JS "Download" means that a download dialog is triggered in the browser. So this is "Fetch".